docs/userguide/campaigns.md

```diff
@@ -1,25 +1,146 @@
-* **Numerical tolerance**: This is a flag for forcing numerical measurements to be within a pre-defined tolerance. Note that the setting of the tolerances is controlled as a part of the respective parameter.
-* **Previously done batches and fits** In case that a campaign builds upon previously condiucted experiments, it is possible to provide the number of previously done batches and fits.
 
-## How to use campaigns
+~~~python
+from baybe import Campaign
 
-We have a detailed example explaining how to use the ``Campaign`` object in our examples, so please see [here](./../../examples/Basics/campaign) for all necessary information.
\ No newline at end of file
+campaign = Campaign(
+    searchspace=searchspace,  # Required
+    objective=objective,  # Required
+    strategy=strategy,  # Optional
+    numerical_measurements_must_be_within_tolerance=boolean,  # Optional
+)
+~~~
+
+```{attention}
+Note that we currently also expose other fields via the constructor. This is
+only temporary, and the corresponding fields should be ignored.
+```
+
+### Creating a campaign via a JSON config
+
+It is also possible to specify a `Campaign` via a configuration string and using the
+function [`Campaign.from_config`](baybe.campaign.Campaign.from_config).
+The specification of config files as well as what they contain are automatically
+derived from the class structure.
+Furthermore, a config file can be validated using 
+[Campaign.validate_config](baybe.campaign.Campaign.validate_config).
+For more details and a full exemplary config, we refer to the corresponding
+[example](./../../examples/Serialization/create_from_config).
+
+## Getting recommendations
+
+### Basics
+
+```{attention}
+Obtaining recommendations resp. adding measurements using the `recommend` resp.
+`add_measurements` functions is the only safe way to inform a `Campaign` object about
+new measurements. These functions update the necessary metadata that is crucial for the
+proper execution of a campaign. It is important to rely on these functions to maintain
+the integrity and reliability of the campaign's execution.
+```
+
+To obtain a recommendation for the next experiment, we can query the campaign and use
+the [`recommend`](baybe.campaign.Campaign.recommend) function. The function takes only
+the `batch_quantity` keyword, specifying the desired size of
+the batch of experiments to be conducted.
+
+~~~python
+rec = campaign.recommend(batch_quantity=3)
+~~~
+
+The `recommend` function returns a `DataFrame` with `batch_quantity` many rows, each 
+representing a set of parameters from the search space.
+
+```{important}
```

Comment:
I know what you tried to convey in this block and it is indeed a very important issue that needs to be mentioned. Hence, having it a in a block is a good idea. However, the current version mixes up a few different things and is also not 100% precise at some places.

First, one thing that is incorrect: _"In this case, the recommendations are made independently of
 each"_ That's just not true, because even in the 1-by-1 scheme, the recommendations are causally linked through the campaign and thus they are dependent. However, perhaps we don't even need to fix it because I think the block should mainly convey a different aspect anyway (see below), and this paragraph might disappear.

What we actually need to convey here is that recommendations are always tailored to the specific requested batch size. That is, if you want 5 experiments, you should EXACTLY query 5, instead of querying 6 and then taking only the top 5. This is because that first batch might be structured completely differently than the second one, due to being batch-optimized. 

The point that you describe instead is also valid, but it's actually completely irrelevant from a user perspective. The reason is that whether the user goes via batch or via 1-by-1 is typically not their choice, or at least they are no direct alternatives to each other. Let me explain the way I see it:

If a user needs X recommendations where X > 1 (i.e. batch setting), they either have the luxury to do the experiments 1-by-1 or not, depending on their time constraints. If they can, then 1-by-1 is certainly going to be better in most cases (I'm not sure, perhaps there is even some formal information-theoretic proof for this???) because the second recommendation will have one additional datapoint for training. However, if they have time constraints, then they are automatically in the batch setting and must query everything at once. That is, they don't really have a choice between the two "alternatives". You know what I mean?

Sorry for the wall of text. If unclear, we can discuss in a meeting. So long story short: your text is not wrong (apart from that one piece) but focuses on the wrong aspect, in my opinion 😃

Reply:
Don't matter the wall of text. Still, since you are the expert on this topic and since I think this would require several other iterations, would you mind to just change this block as you see fit?

Reply:
Hey @AVHopp. Done, have added a few more commits! Please do me a favor and don't just "copy-paste" the text into your fork but use a proper cherry-pick for the additional commits. This both avoids copy-paste issues and also preserves the authorship of the commits for future reference 👍🏼

Reply:
Don't get that point - you are listed as the author of the commits that you did, so what do you mean here? Are you talking about the text passages that I used? These were only posted in comments here, so how should I have cherry-picked them?